llmfoundry/cli/cli.py

```diff
@@ -25,5 +27,67 @@ def train(
     train_from_yaml(yaml_path, args_list)
 
 
+@app.command(name='convert_dataset_hf')
+def convert_dataset_hf_cli(
+    dataset: str = typer.Option(..., '--dataset',
+                                help='Name of the dataset'),  # type: ignore
+    data_subset: Optional[str] = typer.Option(
+        None,
+        '--data_subset',
```

Comment:
seems like this is unnecessary if you aren't changing the name of your parameter? See https://typer.tiangolo.com/tutorial/options/name/

Reply:
Can you update to remove this arg for all the parameters here?